common/types.go

```diff
@@ -218,19 +218,15 @@ var (
 )
 
 func init() {
-	locationToPrefixRange["prime"] = NewRange(0, 9)
-	locationToPrefixRange["cyprus"] = NewRange(10, 19)
-	locationToPrefixRange["cyprus1"] = NewRange(20, 29)
-	locationToPrefixRange["cyprus2"] = NewRange(30, 39)
-	locationToPrefixRange["cyprus3"] = NewRange(40, 49)
-	locationToPrefixRange["paxos"] = NewRange(50, 59)
-	locationToPrefixRange["paxos1"] = NewRange(60, 69)
-	locationToPrefixRange["paxos2"] = NewRange(70, 79)
-	locationToPrefixRange["paxos3"] = NewRange(80, 89)
-	locationToPrefixRange["hydra"] = NewRange(90, 99)
-	locationToPrefixRange["hydra1"] = NewRange(100, 109)
-	locationToPrefixRange["hydra2"] = NewRange(110, 119)
-	locationToPrefixRange["hydra3"] = NewRange(120, 129)
+	locationToPrefixRange["cyprus1"] = NewRange(0, 29)
```

Comment:
We should make these more even, by removing one prefix from Cyprus1 (29) and adding one to Hydra3 (27)

Reply:
https://github.com/dominant-strategies/go-quai/issues/823